### PR TITLE
Added id's to each button.

### PR DESCRIPTION
**Pull request by MHOAKEY**

### CODE REVIEW

**Review by oakeef — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -9,19 +9,19 @@
   </head>
   <body>
     <div>
-      <button class="gameSpace"></button>
-      <button class="gameSpace"></button>
-      <button class="gameSpace"></button>
+      <button class="gameSpace" id="1"></button>
```
Comment (oakeef) [inline index.html:12]: you could be more specific with the ids if you wanted to. a single number doesn't really tell much but also it's a grid of 9 boxes so no big deal either way

**Review by kfoley123 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (MHOAKEY) [reply index.html:12]: Makes sense. I wanted to keep it simple but I can see how more specific could benefit.

Comment (MHOAKEY) [reply index.html:12]: I want to refer to the innerHTML of id's to compare them and use that to solve for a victory.

Comment (oakeef) [reply index.html:12]: that is reasonable. you could also just use the indices of the array of HTML elements you get back when you use 
`querySelectorAll(".gameSpace")`

**Pull request merged**